### ISSUE TITLE
Ruby Sass is deprecated and will be unmaintained as of 26 March 2019

### ISSUE THREAD

**Issue by dmarcoux** [openSUSE/open-build-service#6263]
I stumbled onto this while running *bundler* locally. We should take action soon.

```
Ruby Sass is deprecated and will be unmaintained as of 26 March 2019.

* If you use Sass as a command-line tool, we recommend using Dart Sass, the new
  primary implementation: https://sass-lang.com/install

* If you use Sass as a plug-in for a Ruby web framework, we recommend using the
  sassc gem: https://github.com/sass/sassc-ruby#readme

* For more details, please refer to the Sass blog:
  http://sass.logdown.com/posts/7081811
```

**Comment by coolo** [comment]
We need https://github.com/twbs/bootstrap-rubygem/commit/7c6056eebb7a9fd8cac7845ca31bf62a81627df2 to be released

the rest is https://github.com/openSUSE/open-build-service/pull/6264

**Comment by coolo** [comment]
Fixed in 99259053cc2dba78cc0ef110b7b6c0bdd07b5521
